Python/Wikipedia_Footballer_Scraper/main.py

```diff
@@ -0,0 +1,107 @@
+from bs4 import BeautifulSoup
+import requests
+
+
+def get_player_name():
+    """
+    Takes Football player's name input from the user and
+    returns a string suitable to concatenate in the wikipedia URL.
+    """
+
+    player = input('\nEnter the name of the Football player: ').split(' ')
+    player_name = ''
```

Comment:
Here, a better approach will be use something like `join` function. Essentially since you are capitalizing all the elements in the `player` array and creating a string that has all player names with spacing=1.

Can you try refactoring this? 
Refer this for help : https://www.oreilly.com/library/view/javascript-the-definitive/0596101996/re08.html